src/flag.h

```diff
@@ -53,6 +58,7 @@ class json_flag
     private:
         const std::string id_;
         std::string info_;
+        std::string restriction_;
```

Comment:
Please use `class translation` for translatable strings (And if you don't mind, update `info_` to use `class translation` too).